Day-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js

```diff
@@ -1,2 +1,10 @@
 import React from 'react';
 import './Button.css';
+
+const ActionButton = (props) => {
+    return (
+        <div class={props.buttonStyle}>{props.text}</div>
```

Comment:
since it's one line, would be best as `return <div class={props.buttonStyle}>{props.text}</div>`

meaning: no `()`s